src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.managed.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#if !NET461
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.PublicMethodArgumentsShouldBeCheckedForNull' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: AddView ## Method file: VirtualizingPanelLayout.managed.cs ## Method line: 613,2 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode ](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5187634)